src/test/scala/nl.knaw.dans.easy.deposit/docs/AgreementsSpec.scala

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (C) 2018 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.deposit.docs
+
+import nl.knaw.dans.easy.deposit.TestSupportFixture
+import nl.knaw.dans.easy.deposit.docs.JsonUtil.InvalidDocumentException
+import nl.knaw.dans.easy.deposit.docs.dm.PrivacySensitiveDataPresent
+import org.joda.time.DateTime
+
+import scala.util.{ Failure, Success }
+
+class AgreementsSpec extends TestSupportFixture {
+
+  "apply" should "complain about not accepted deposit agreement" in {
+    AgreementsXml(
+      "user",
+      DateTime.now,
+      DatasetMetadata().copy(
+        acceptDepositAgreement = false,
+        privacySensitiveDataPresent = PrivacySensitiveDataPresent.no
+      )
+    ) should matchPattern { case Failure(InvalidDocumentException(_, e)) if e.getMessage == "Please set AcceptDepositAgreement" => }
+  }
+  it should "complain about not specifying the presence of privacy sensitive data" in {
+    AgreementsXml(
+      "user",
+      DateTime.now,
+      DatasetMetadata().copy(
+        acceptDepositAgreement = true,
+        privacySensitiveDataPresent = PrivacySensitiveDataPresent.unspecified
+      )
+    ) should matchPattern { case Failure(InvalidDocumentException(_, e)) if e.getMessage == "Please set PrivacySensitiveDataPresent" => }
+  }
+  "schema validation" should "succeed" in {
+    AgreementsXml(
+      "user",
+      DateTime.now,
+      DatasetMetadata().copy(
+        acceptDepositAgreement = true,
+        privacySensitiveDataPresent = PrivacySensitiveDataPresent.no
+      )
+    ).flatMap(AgreementsXml.validate) should matchPattern { case Success(_) => }
```

Comment:
@lindareijnhoudt 

why does this fail with

    SAXParseException; lineNumber: 1; columnNumber: 340; cvc-elt.1: Cannot find the declaration of element 'agr:agreements'

on
```xml
<agr:agreements xsi:schemaLocation="http://easy.dans.knaw.nl/schemas/bag/metadata/agreements http://easy.dans.knaw.nl/schemas/bag/metadata/agreements/2018/05/agreements.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:dcterms="http://purl.org/dc/terms/" xmlns:agr="http://easy.dans.knaw.nl/schemas/bag/metadata/agreements">
  <agr:licenseAgreement>
    <agr:depositorId>user</agr:depositorId>
    <dateAccepted>2018-03-22T21:43:01.000+01:00</dateAccepted>
    <agr:licenseAgreementAccepted>true</agr:licenseAgreementAccepted>
  </agr:licenseAgreement>
  <agr:personalDataStatement>
    <agr:signerId>user</agr:signerId>
    <agr:dateSigned>2018-03-22T21:43:01.000+01:00</agr:dateSigned>
    <agr:containsPrivacySensitiveData>false</agr:containsPrivacySensitiveData>
  </agr:personalDataStatement>
</agr:agreements>
```